web/src/user_card_popover.js

```diff
@@ -24,7 +24,7 @@ import * as narrow from "./narrow";
 import * as overlays from "./overlays";
 import {page_params} from "./page_params";
 import * as people from "./people";
-import * as popovers from "./popovers";
+import * as popovers_module from "./popovers";
```

Comment:
We try to avoid this sort of aliasing; it makes it a lot harder to find call points for something with say `git grep popovers.foo`. I think better would be to just call your new object `user_card_popovers`.

Reply:
OK I think that specific proposal is madness; we'd get `user_card_popover.user_card_popovers.manage_menu.is_open`.

I think a good API for this sort of thing is to export a class and just instantiate that class for each popover. E.g. we'd have `export manage_menu = new PopoverMenu(...)` and then the above call to check things would be `user_card_popover.manage_menu.is_open`?

Or we could just not worry too much about having semi-duplicate `hide` functions.